terraform/etl/48-lambda-gov-notify-ingestion.tf

```
@@ -120,3 +128,48 @@ resource "aws_iam_role_policy_attachment" "gov_notify_lambda_secret_access" {
   role       = aws_iam_role.housing_gov_notify_ingestion[0].name
   policy_arn = aws_iam_policy.gov_notify_lambda_secret_access[0].arn
 }
+
+
+module "gov-notify-ingestion-housing-repairs" {
+  count                          = !local.is_production_environment ? 1 : 0
+  source                         = "../modules/aws-lambda"
+  tags                           = module.tags.values
+  lambda_name                    = "govnotify_api_ingestion_repairs"
+  identifier_prefix              = local.short_identifier_prefix
+  handler                        = "main.lambda_handler"
+  lambda_artefact_storage_bucket = module.lambda_artefact_storage.bucket_id
```

in the etl state you need to refer to this bucket like this:
```suggestion
  lambda_artefact_storage_bucket = module.lambda_artefact_storage_data_source.bucket_id
```